### ISSUE TITLE
Syncing Epichrome to other workstations

### ISSUE THREAD

**Issue by ylluminate** [dmarmor/epichrome#126]
Hey, I got to thinking about your previous work to detect all Epichrome instances to make proper updates.

Any suggestions for making a script with rsync that will detect all Epichrome App instances and then will sync out the app and configs to a destination?  It would be nice to have something like:
`epichrome app_sync  <all | app_name> dest_username@dest_host`

So far I've been doing, but it's a bit laborious and awkward:
`scp /Applications/Epichrome_app_name.app "username@host:/Applications/"; scp -prq "/Users/username/Library/Application Support/Epichrome/Apps/Epichrome_app_name" "username@host/Application\ Support/Epichrome/Apps/"`

**Comment by dmarmor** [comment]
Unfortunately, I've never really used rsync, so I'm not sure how much I can help with this. What I can tell you is that if you sync the app bundle itself and the contents of the corresponding Application Support directory (as it appears you're doing  with your scp command?) that should give you a proper new instance with all the same settings.

The only thing I noticed is that the name of the directory in your Library folder will not necessarily have the same name as the app--it actually uses the app ID, which can be the name of the app, but may be a hash if the app name has spaces or is similar or identical to another app.

I hope that helps?

**Comment by ylluminate** [comment]
Okay thanks, so I think the most useful thing would be this: is there a way to identify all Epichrome apps from an Applications folder with each ones' accompanying `Application Support/Epichrome` folder?

**Comment by ylluminate** [comment]
@dmarmor so I found something peculiar.  When I copy apps over now the apps are not retaining their settings, neither are the retaining settings even after starting and then customizing and quitting the Epichrome apps!  Extremely strange behavior... Any thoughts?

**Comment by ylluminarious** [comment]
@dmarmor I am having a similar issue. Epicrome apps simply lose all settings and extensions when transferred to another machine. Even when copying both the app bundle and the settings under `~/Library/Application Support/Epichrome/Apps`.

**Comment by ylluminate** [comment]
So @dmarmor I spent a little time on this synchronization issue and I got it working... kind of.

It turns out that every time I push an app to another workstation, Chrome via Epichrome appears to NOT allow extensions (and themes) that were installed on another workstation to work on the new workstation.  Bookmarks & History and all are copied and show up fine, but when I go into settings it warns me that it reset some settings and then when I look into the `~/Library/Application Support/Epichrome/Apps/APPNAME/Default/Extensions/` it suddenly empties out right before my eyes...

Any idea of how I could stop Chrome from doing this?  I'm guessing there's some kind of signature issue between workstations that craps all over the place, but I'm not yet sure...
